product/reports/170_Configuration Management - Containers/010_Nodes by Capacity.yaml

```diff
@@ -1,6 +1,5 @@
 title: Nodes By Capacity
 rpt_group: Custom
```

Comment:
The previous report patch went in too quickly. @dclarizio @h-kataria can you check if there's anything else to adjust? For example `rpt_group`. Thanks.